docs/manuals/source/DeveloperGuide/netprotocol.rst

```diff
@@ -245,3 +245,20 @@ The following diagram shows the activity on a Bareos TCP server (i.e. a |dir| or
   :Handle client message;
   end
 
+
+Debugging
+---------
+
+Developers who want to debug the message flow can use the network dump feature. This feature is only for specialists and is **not intended for use in a production system**.
+
+This feature is only for developers and therefore no help will be provided on the command line. The arguments however can be mixed with the standard arguments.
+
+.. code-block:: shell-session
+   :caption: Network Dump Command Line Options
+
+   Usage: [bconsole|bareos-dir|bareos-fd|bareos-sd]
+          -z <filename>    name and path of the dumpfile
+                           (setting this switches on the network dump)
+          -zp              use the plantuml notation
+          -z <stackdepth>  for additional stacktrace, stackdepth=-1: print all frames,
```

Comment:
same short-option for filename and stackdepth, is this correct?

Reply:
Works as desired.